src/libraries/System.Runtime/tests/System/Reflection/ReflectionCacheTests.cs

```diff
@@ -40,17 +68,47 @@ public void GetMethod_MultipleCalls_ClearCache_DifferentObjects(bool justSpecifi
         {
             Action<Type[]> clearCache = GetClearCacheMethod();
 
-            MethodInfo mi1 = typeof(ReflectionCacheTests).GetMethod(nameof(GetMethod_MultipleCalls_ClearCache_DifferentObjects));
+            MethodInfo mi1 = s_type.GetMethod(nameof(GetMethod_MultipleCalls_ClearCache_DifferentObjects));
+            PropertyInfo pi1 = s_type.GetProperty(nameof(Property));
+            FieldInfo fi1 = s_type.GetField(nameof(Field1));
+            EventInfo ei1 = s_type.GetEvent(nameof(Event1));
+            ConstructorInfo ci1 = s_type.GetConstructor(Type.EmptyTypes);
+            ParameterInfo pai1 = mi1.GetParameters()[0];
             Assert.NotNull(mi1);
+            Assert.NotNull(pi1);
+            Assert.NotNull(fi1);
             Assert.Equal(nameof(GetMethod_MultipleCalls_ClearCache_DifferentObjects), mi1.Name);
+            Assert.Equal(nameof(Property), pi1.Name);
+            Assert.Equal(nameof(Field1), fi1.Name);
 
             clearCache(justSpecificType ? new[] { typeof(ReflectionCacheTests) } : null);
+            Assert.True(HotReloadDeltaApplied());
 
-            MethodInfo mi2 = typeof(ReflectionCacheTests).GetMethod(nameof(GetMethod_MultipleCalls_ClearCache_DifferentObjects));
+            MethodInfo mi2 = s_type.GetMethod(nameof(GetMethod_MultipleCalls_ClearCache_DifferentObjects));
+            PropertyInfo pi2 = s_type.GetProperty(nameof(Property));
+            FieldInfo fi2 = s_type.GetField(nameof(Field1));
+            EventInfo ei2 = s_type.GetEvent(nameof(Event1));
+            ConstructorInfo ci2 = s_type.GetConstructor(Type.EmptyTypes);
+            ParameterInfo pai2 = mi2.GetParameters()[0];
             Assert.NotNull(mi2);
+            Assert.NotNull(pi2);
+            Assert.NotNull(fi2);
             Assert.Equal(nameof(GetMethod_MultipleCalls_ClearCache_DifferentObjects), mi2.Name);
+            Assert.Equal(nameof(Property), pi2.Name);
+            Assert.Equal(nameof(Field1), fi2.Name);
 
             Assert.NotSame(mi1, mi2);
```

Comment:
This block of `NotSame` and `Equal` is different enough that I think it warrants a comment above the block inidcating what we are checking.

Reply:
Right